apps/datahub/src/app/dataset-view/record-view.component.ts

```diff
@@ -8,9 +9,14 @@ import { MdViewFacade } from '@geonetwork-ui/feature/search'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RecordViewComponent {
-  constructor(public facade: MdViewFacade) {}
+  constructor(
+    public facade: MdViewFacade,
+    private mapManager: MapManagerService
+  ) {}
 
   onTabIndexChange(index: number): void {
-    console.log(index)
+    if (index === 2) {
+      setTimeout(() => this.mapManager.map.updateSize(), 0)
```

Comment:
this brings back memories... Ideally the map presentation component should be able to do that itself by using e.g. [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)